server/server.go

```diff
@@ -144,21 +149,21 @@ func RootHandler(ctx context.Context, ac auth.AccessController, trust signed.Cry
 	r := mux.NewRouter()
 	r.Methods("GET").Path("/v2/").Handler(authWrapper(handlers.MainHandler))
 
-	r.Methods("POST").Path("/v2/{imageName:.*}/_trust/tuf/").Handler(createHandler(_serverEndpoint{
+	r.Methods("POST").Path("/v2/{imageName:[^*]+}/_trust/tuf/").Handler(createHandler(_serverEndpoint{
```

Comment:
@cyli it seems `([^*]|.{2,})` will  just match out the first character if it is not `*`. It will not use the 2nd alternative unless the string is begin with `*`. For example `abcd`, only the `a` will be matched out, though `*abcd` will match the whole string.